tests/fixture/pageObject/PaymentStep.ts

```diff
@@ -0,0 +1,23 @@
+import { Page } from '@playwright/test';
+
+import { AssertionsHelper, CheckoutPresets, PlaywrightHelper } from '.';
+
+export default class PaymentStep {
+    assertions: AssertionsHelper;
+    isReplay: boolean;
+    playwright: PlaywrightHelper;
```

Comment:
This can remain `private`. I can see it is used in tests like this 
```ts
paymentStep.playwright.serveFile({
    routePath: '/checkout/payment/hosted-field?*',
    filePath: './tests/sampleTests/support/hostedField.ejs',
})
```
Instead, that bit of detail can hidden away from tests and be done as a part of the initialisation process of `PaymentStep` (i.e.: `goto`)

Reply:
Sure, will move them to the initialisation.